exports/exports.go

```diff
@@ -327,8 +325,8 @@ func DBExportToAPI(payload models.ExportPayload) ExportPayload {
 		}
 
 		if source.SourceError != nil {
-			newSource.Message = source.SourceError.Message
-			newSource.Code = source.SourceError.Code
+			newSource.Message = source.Message
+			newSource.Code = source.Code
 		}
 
```

Comment:
Hmm...I'm not sure about this change.

Oh!  This works due to struct embedding!!   Looks fine!